crates/services/gas_price_service/src/fuel_gas_price_updater/fuel_core_storage_adapter/metadata_tests.rs

```diff
@@ -36,25 +30,6 @@ fn database() -> StorageTransaction<InMemoryStorage<GasPriceColumn>> {
     InMemoryStorage::default().into_transaction()
 }
 
-#[tokio::test]
-async fn get_metadata__can_get_most_recent_version() {
```

Comment:
Why do we want to remove this test?

Reply:
No. They are testing different functions. It doesn't matter if they are redundant, the point isn't coverage.

Reply:
okay

Reply:
addressed in 62343d76a8